packages/server/src/index.ts

```diff
@@ -1793,14 +1800,15 @@ export class App {
                 query.andWhere('chat_message.sessionId = :sessionId', { sessionId })
             }
 
-            // set date range
             query.andWhere('chat_message.createdDate BETWEEN :fromDate AND :toDate', {
-                fromDate: fromDate ?? new Date().setMonth(new Date().getMonth() - 1),
-                toDate: toDate ?? new Date()
+                fromDate: fromDate ?? new Date(new Date().setHours(0, 0, 0, 0)),
+                toDate: toDate ?? new Date(new Date().setHours(23, 59, 59, 999))
             })
-            // sort
+
             query.orderBy('chat_message.createdDate', sortOrder === 'DESC' ? 'DESC' : 'ASC')
 
+            const [sql, parameters] = query.getQueryAndParameters()
```

Comment:
these 2 `[sql, parameters]` are not being used?

Reply:
Yes. Should be removed. Should have run lint.